examples/morpheusvm/auth/bls.go

```diff
@@ -21,7 +21,8 @@ var _ chain.Auth = (*BLS)(nil)
 
 const (
 	BLSComputeUnits = 10
-	BLSSize         = bls.PublicKeyLen + bls.SignatureLen
+	BLSCompressSize	= bls.PublicKeyLen * 2	
```

Comment:
We should just use the byte length of the serialized key here?

What use is the non-serialized key length?

Reply:
I'm not sure `PublicKeyLen` isn't this already: https://github.com/ava-labs/avalanchego/blob/7623ffd4be915a5185c9ed5e11fa9be15a6e1f00/utils/crypto/bls/public.go#L12C41-L12C41